src/hooks/useQuoteManager.js

```diff
@@ -1,14 +1,38 @@
 import axios from 'axios';
 import { useState } from 'react';
-
-const AUTH_TOKEN = `bearer ${process.env.REACT_APP_API_TOKEN}`;
+import qs from 'qs';
 
 export function useQuoteManager() {
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState();
   const [searchResults, setSearchResults] = useState();
   const [quoteResult, setQuoteResult] = useState();
   const [applicationSuccess, setApplicationSuccess] = useState();
+  const [token, setToken] = useState();
+
+  async function getToken() {
+    try {
+      const body = {
+        client_id: process.env.REACT_APP_CLIENT_ID,
+        client_secret: process.env.REACT_APP_CLIENT_SECRET,
+        grant_type: 'client_credentials',
+      };
+      const response = await axios({
+        headers: {
+          'Content-type': 'application/x-www-form-urlencoded',
+          client_id: process.env.REACT_APP_CLIENT_ID,
+          client_secret: process.env.REACT_APP_CLIENT_SECRET,
+          grant_type: 'client_credentials',
+        },
+        method: 'POST',
+        data: qs.stringify(body),
```

Comment:
Removed the library, and used a template string. the output is a query string because we're using
'Content-type': 'application/x-www-form-urlencoded'